src/gantry/widgets.py

```
@@ -45,6 +47,110 @@ def __init__(self, *args, **kwargs):
+        )
+        if shift:
+            if existing_pos is not None:
+                idx, rev = self._sort_columns[existing_pos]
+                self._sort_columns[existing_pos] = (idx, not rev)
+            else:
+                if len(self._sort_columns) >= 3:
+                    # Drop least-significant key to preserve primary sort intent
+                    self._sort_columns.pop()
+                self._sort_columns.append((col_idx, False))
+        else:
+            if existing_pos == 0 and len(self._sort_columns) == 1:
```

Ruff (0.15.12)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)